### DISCUSSION TITLE
Reg: plot_decision_regions  : how to speed up

### DISCUSSION THREAD

**Discussion by sternparky** [rasbt/mlxtend#801] [Q&A]
To whom it may concern,

I'd like to speed up the plot decision regions by plotting "roughly" without high resolution or fancy looking.

Is there an easy way?

Thanks.

**Comment by rasbt** [comment]
Hi there,
there used to be a `res` argument but it is currently deprecated and not doing anything. Users usually only used to increase resolution. We replaced this with an automatic way to determine the resolution.

Right now, there is no way to make "rougher" plots. However, a "`low_resolution`" or "`fast_plot`" argument could be added in the future. I will add that as a suggested future enhancement to the issue list (#804)
